lib/ajv.d.ts

```diff
@@ -1,7 +1,9 @@
-declare function ajv (options?: ajv.Options): ajv.Ajv;
-
-declare namespace ajv {
-  interface Ajv {
+  export default class Ajv {
```

Comment:
This breaks because this library is not an `export default`. You can read about what `export default` and `export =` means https://www.typescriptlang.org/docs/handbook/modules.html